langchain/chains/sql_database/base.py

```diff
@@ -96,36 +105,80 @@ def _call(
         table_info = self.database.get_table_info(table_names=table_names_to_use)
         llm_inputs = {
             "input": input_text,
-            "top_k": self.top_k,
+            "top_k": str(self.top_k),
             "dialect": self.database.dialect,
             "table_info": table_info,
-            "stop": ["\nSQLResult:"],
+            "stop": ["\n", "SQLResult:"],
```

Comment:
i dont think we want \n in here - i think SQL can generate new lines in a valid day

Reply:
ok, fix coming for this... I saw a few cases that I perhaps overfit for with my local model... reverting this per your feedback